src/fu-usb-backend.c

```diff
@@ -1,5 +1,6 @@
 /*
- * Copyright 2021 Richard Hughes <richard@hughsie.com>
+ * Copyright 2011 Richard Hughes <richard@hughsie.com>
```

Comment:
2011-2024

Reply:
I think the copyright scanner will get upset if I use an end date.

Reply:
That's fine then just leave em as is